cli/internal/helm/charts/edgeless/constellation-services/charts/ccm/templates/aws-daemonset.yaml

```diff
@@ -0,0 +1,63 @@
+{{ if eq .Values.csp "AWS" }}
+apiVersion: apps/v1
+kind: DaemonSet
```

Comment:
Not sure how this would influence ease of use, but, since there is a lot of equality between the CCM deployment of different CSPs, we could merge all ccm daemonsets into a single file and then just set the required values (image, command, CSP specific mounts) conditionally

Reply:
I am happy to change this if the majority prefers it that way, but I would prefer to keep them separate. It seems to me that the rendered file is a bit easier to predict this way, over having a lot of if-else statements.